tsl/src/continuous_aggs/create.c

```diff
@@ -1663,9 +1748,19 @@ add_aggregate_partialize_mutator(Node *node, AggPartCxt *cxt)
 		if (cxt->ignore_aggoid == ((Aggref *) node)->aggfnoid)
 			return node; /*don't process this further */
 
+		bool old_inside_aggref = cxt->inside_aggref;
+		if (!old_inside_aggref)
+			cxt->inside_aggref = true;
```

Comment:
It looks like `cxt->inside_aggref` is not used in `add_partialize_column` so it's probably unnecessary to set it and then unset it.

Reply:
I probably created them initially during recursion, but at this point it seems like the code doesn't recurse into anything anymore, so you are right, they should be removed.